README.md

```diff
@@ -179,11 +179,6 @@ Browser Support
 
 ###### * Safari may function correct, but we are not actively testing with it
 
-Discussion
-------------
-
-Join our [discord server](https://discord.gg/bPSWD73)
-
 <a name="contributions"/> 
```

Comment:
Could not understand the relevance of this line?

Reply:
This is an anchor which allows linking directly to the contributions section of the page (the section following Discussion).

Reply:
Seem like link to sections just work fine without an explicit anchor tag.  I have tried removing the the tag in one of the test branches (temp-remove-hasteresolver) and link to the contributions section created at the end of first section just works fine.

Reply:
I'm not sure.  Could you elaborate on what you're seeing?
I'm seeing http://schrodinger.github.io/fixed-data-table-2/#contributions work as I expect, 
while http://schrodinger.github.io/fixed-data-table-2/#discussion opens to just below the start of discussion, 
and http://schrodinger.github.io/fixed-data-table-2/#getting%20started just opens the top of the page.

Either way, this may be nice to investigate and cleanup, but let's do via a separate issue.

Reply:
For me  http://schrodinger.github.io/fixed-data-table-2/#contributions opens just above **discussion**.  http://schrodinger.github.io/fixed-data-table-2/#discussion too opens just above discussion and http://schrodinger.github.io/fixed-data-table-2/#getting%20started just opens the top of the page.

Yup we could do the clean up if required latter.  I was just curious to understand whats going on.